File: src/main/java/com/uisrael/medical_service/model/UserDTO.java
package com.uisrael.medical_service.model;

import com.uisrael.medical_service.entities.Dispensary;
import com.uisrael.medical_service.entities.Role;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;


import java.util.ArrayList;
import java.util.Date;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserDTO {

    private Long id;
    private Date dateCreate = new Date();
    private String identification;
    private String name;
    private String lastName;
    private String username;
    private String email;
    private String password;
    private Role role;
    private Integer status;

}

File: FrontEnd/MedicApp/DTO's/UserDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MedicApp.DTO_s
{
    public class UserDTO
    {
        public long id { get; set; }

        public long dateCreate { get; set; }
        public DateTime DateCreateAsDateTime => DateTimeOffset.FromUnixTimeMilliseconds(dateCreate).DateTime;
        public string identification { get; set; }
        public string name { get; set; }
        public string lastName { get; set; }
        public string username { get; set; }
        public string email { get; set; }
        public string password { get; set; }
        public RoleDTO role { get; set; } 
        public int status { get; set; }

        public string StatusText => status != 1 ? "Inactivo" : "Activo";

        public string StatusColor => status != 1 ? "Red" : "Green";

    }
}
